roles/KubevirtNodeLabeller/templates/kubevirt-node-labeller-ds.yaml.j2

```diff
@@ -44,7 +44,7 @@ spec:
               name: cpu-config
 
         - name: libvirt
-          image: {{ ssp_registry | default('kubevirt') }}/{{ libvirt_image }}:{{ libvirt_tag }}
+          image: {{ lookup('env','VIRT_LAUNCHER_IMAGE')| default(virt_launcher_image, true) }}
```

Comment:
in retrospect I think it would be better to use two env vars, one for the libvirt_image and one for the libvirt_tag. I think it should look like
```
 image: {{ ssp_registry | default('kubevirt') }}/{{ lookup('env','VIRT_LAUNCHER_IMAGE')| default(libvirt_image, true) }}:{{ lookup('env','VIRT_LAUNCHER_TAG')| default(libvirt_tag, true) }}
```

Reply:
@tiraboschi can the script which builds CSV replace only e.g. part of value?

Reply:
Now it's replacing only the TAG

Reply:
Replacing the tag would actually address my concern above! we can keep the ssp_registry variable and get the right tag value from HCO.
@ksimon1 , do you think you can update the PR to make this happen?